client/pages/detail.jsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import Navbar from '../components/navbar';
+import Loading from './loading';
+
+class detail extends React.Component {
+  constructor(props) {
+    const device = window.innerWidth < 768 ? 'small' : 'regular';
+    super(props);
+    this.state = {
+      intervalId: null,
+      carouselIndex: 0,
+      game: null,
+      screenshots: null,
+      device
+    };
+  }
+
+  componentDidMount() {
+    const gameId = this.props.id;
+
+    fetch(`http://localhost:3000/api/game/${gameId}`)
```

Comment:
Try to leave out the host name and port number, since we will eventually go live online and the app won't be on localhost anymore
```js
fetch(`/api/game/${gameId}`)
```